modular_skyrat/modules/vore/code/datums/vore_prefs.dm

```diff
@@ -0,0 +1,584 @@
+//original code by: Seris02
+
+/datum/vore_prefs
```

Comment:
This datum has dubious utility considering the TG preferences rework. I'd seriously consider moving these preferencees to /datum/preference options.

Reply:
no